code/include/playfab_c/PlayFabClientApi_c.h

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include "httpClient/pal.h"
+#include "httpClient/async.h"
+#include "PlayFabClientDataModels_c.h"
+
+/// <summary>
+/// Initialize PlayFab global state
+/// </summary>
+HRESULT PlayFabInitialize(
+    _In_ const char* titleId
+) noexcept;
+
+/// <summary>
+/// Cleanup PlayFab global state. Shouldn't be called while async APIs are still running.

```

Comment:
>Shouldn't be called while async APIs are still running. [](start = 34, length = 55)

comment still true?

Reply:
Need to discuss this more.  Will loop in ATG and others and try to get more feedback

---
In reply to: [590795173](https://github.com/jplafonta/XPlatCppSdk/pull/2#discussion_r590795173) [](ancestors = 590795173)